core/tests/vm-benchmark/README.md

```diff
@@ -9,35 +9,22 @@ benchmarks, however.
 There are three different benchmarking tools available:
 
 ```sh
-cargo bench --bench criterion
-cargo bench --bench diy_benchmark
+cargo bench --bench oneshot
+cargo bench --bench batch
 cargo +nightly bench --bench iai
 ```
 
-Criterion is the de-facto microbenchmarking tool for Rust. Run it, then optimize something and run the command again to
-see if your changes have made a difference.
+`oneshot` and `batch` targets use Criterion, the de-facto standard micro-benchmarking tool for Rust. `oneshot` measures
+VM performance on single transactions, and `batch` on entire batches of up to 5,000 transactions. Run these benches,
+then optimize something and run the command again to see if your changes have made a difference.
 
-The DIY benchmark works a bit better in noisy environments and is used to push benchmark data to Prometheus
-automatically.
+IAI uses cachegrind to simulate the CPU, so noise is completely irrelevant to it, but it also doesn't measure exactly
+the same thing as normal benchmarks. You need valgrind to be able to run it.
 
-IAI uses cachegrind to simulate the CPU, so noise is completely irrelevant to it but it also doesn't measure exactly the
-same thing as normal benchmarks. You need valgrind to be able to run it.
-
-You can add your own bytecodes to be benchmarked into the folder "deployment_benchmarks". For iai, you also need to add
-them to "benches/iai.rs".
+You can add new bytecodes to be benchmarked into the [`bytecodes`](src/bytecodes) directory and then add them to the
+`BYTECODES` constant exported by the crate.
 
 ## Profiling (Linux only)
 
 You can also use `sh perf.sh bytecode_file` to produce data that can be fed into the
 [firefox profiler](https://profiler.firefox.com/) for a specific bytecode.
-
-## Fuzzing
```

Comment:
AFAIU, this section is irrelevant for newer VM versions.

Reply:
Yes, fuzzing the old VM is pretty useless because you need to manually edit rust std to get high stability. Fuzzing the new VM to find exceptionally poorly performing inputs may be worthwhile.